wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,395 @@
+---
+tags:
+  - history
+  - leaderboard
+  - rank 1
+  - rank #1
+  - rankings
+  - top player
+---
+
+<!-- Maintainers Note: Please refer to the original pull request for more info on sources at https://github.com/ppy/osu-wiki/pull/4720 -->
+
+# osu! rankings history
+
+The following is a list of players who are recognized to have reached Rank #1 in the [osu! Global Rankings](https://osu.ppy.sh/rankings/osu/performance).
+
+Over the years, the osu! global rankings has used three different systems for ranking players:
+
+- [Ranked Score](/wiki/score#ranked-score) (6 October 2007 – 24 July 2012)
+- [ppv1](/wiki/Performance_points/ppv1) (24 July 2012 – 27 January 2014)
+- [ppv2](/wiki/Performance_points) (27 January 2014 – Present)
+
+Because of this, this list will only include players who reached Rank #1 on the main leaderboard that was used at the time of reaching said rank. *Italicized* dates and numbers indicate that the corresponding player is continuing their reign as osu!'s Rank #1 player either from the previous year or into the following year.
```

Comment:
```suggestion
Because of this, this list only includes players who have reached Rank #1 on the main leaderboard that was used at the time. *Italicized* dates and numbers indicate that the corresponding player is continuing their reign as osu!'s Rank #1 player either from the previous year or into the following year.
```